packages/grant-explorer/src/features/api/utils.ts

```diff
@@ -431,6 +432,45 @@ export function getChainIds(): number[] {
       Number(ChainId.PGN),
     ];
   } else {
-    return Object.values(ChainId).map((chainId) => Number(chainId));
+    return Object.values(ChainId)
+      .map((chainId) => Number(chainId))
+      .filter((id) => !isNaN(id));
```

Comment:
No. ChainId is a typescript enum with values defined as numbers, so unless someone goes and defines a value as something that is not convertible to a number, .e.g "GOERLI", this will work. If someone does this, my _hope_ is that we will catch it in PR review. I'm thinking of other, better ways to do this than typescript enums, as they have a lot of quirks.